webapi/src/test/scala/org/knora/webapi/responders/v2/ValuesResponderV2Spec.scala

```diff
@@ -1780,6 +1780,32 @@ class ValuesResponderV2Spec extends CoreSpec() with ImplicitSender {
       }
     }
 
+    "not create a list value that is a root list node" in {
+      val resourceIri           = "http://rdfh.ch/0001/a-blue-thing"
+      val resourceClassIri      = "http://www.knora.org/ontology/0001/anything#BlueThing".toSmartIri
+      val propertyIri: SmartIri = "http://0.0.0.0:3333/ontology/0001/anything/v2#hasListItem".toSmartIri
+      val valueHasListNode      = "http://rdfh.ch/lists/0001/otherTreeList"
+
+      appActor ! CreateValueRequestV2(
+        CreateValueV2(
+          resourceIri = resourceIri,
+          resourceClassIri = resourceClassIri,
+          propertyIri = propertyIri,
+          valueContent = HierarchicalListValueContentV2(
+            ontologySchema = ApiV2Complex,
+            valueHasListNode = valueHasListNode
+          )
+        ),
+        featureFactoryConfig = defaultFeatureFactoryConfig,
+        requestingUser = anythingUser1,
+        apiRequestID = UUID.randomUUID
+      )
+
+      expectMsgPF(timeout) { case msg: akka.actor.Status.Failure =>
+        assert(msg.cause.isInstanceOf[NotFoundException])
```

Comment:
should this really be a "NotFoundException", seems more like a "BadRequest", right?

Reply:
It has been improved with [58b4551](https://github.com/dasch-swiss/dsp-api/pull/2088/commits/58b4551f7321c27301650478108afac59f56ddec) and now depends of outcome I throw `BadRequestException` or `NotFoundException`.